getting-started-react-esbuild/README.md

```diff
@@ -0,0 +1,205 @@
+# Getting Started with Metaplex and Esbuild
+
+This example sets up a new React app with Metaplex using Esbuild.
+
+This example has been generated using the following steps:
+
+1. **Create a new project using NPM.**
+
+   Esbuild does not has any standard template for React which can be utilized for initializing the project like CRA or Vite. So, we will be creating a empty npm project and setup everything from scratch.
+
+   ```sh
+   mkdir getting-started-react-esbuild
+   cd getting-started-react-esbuild
+   npm init -y
+   ```
+
+   Once empty npm project is setup, we will setup a basic directory structure with mainly `public` and `src` folder and a config file (`build.js`) which will be used for compiling and bundling the react app with esbuild.
+
+   ```
+   getting-started-react-esbuild/
+   ├── public/
+   │   └── index.html
+   ├── src/
+   │   ├── app.jsx
+   │   ├── index.css
+   │   └── index.js
+   ├── build.js
+   └── package.json
+   ```
+
+2. **Install the React, React DOM, EsBuild, Metaplex and the Solana SDKs.**
+
+   ```sh
+   npm install react react-dom esbuild @metaplex-foundation/js @solana/web3.js
+   ```
+
+3. **Install some polyfills.**
+
+   <details>
+     <summary>Why?</summary>
+     Some dependencies of the Metaplex SDK are still relying on NPM packages that are not available in the browser. To make sure that the Metaplex SDK works in the browser, we need to install some polyfills.
+   </details>
+
+   ```sh
+   npm install -D node-stdlib-browser
+   ```
+
+4. **Install Servor for Development**
+
+    <details>
+     <summary>Why?</summary>
+     Esbuild does not include any server that could be used to preview the esbuild built bundle in browser. So we will be using servor for that
+   </details>
+
+   ```sh
+   npm install -D servor
+   ```
+
+5. **Update your Esbuild build file and package.json**.
+
+   Add the following code to `build.js` file we created.
+
+   <details>
+     <summary>Why?</summary>
+     The following code will build and bundle your react app with EsBuild. It will also start a dev server when using in development
```

Comment:
```suggestion
     The following code will build and bundle your react app with EsBuild. It will also start a dev server when used in development.
```